.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,33 @@
+# This workflow will run tests using node and then publish a package to npm when a release is created
+# For more information see: https://help.github.com/actions/language-and-framework-guides/publishing-nodejs-packages
+
+name: Publish Node.js Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  # build:
+  #   runs-on: ubuntu-latest
+  #   steps:
+  #     - uses: actions/checkout@v3
+  #     - uses: actions/setup-node@v3
+  #       with:
+  #         node-version: 16
+  #     - run: npm ci
+  #     - run: npm test
```

Comment:
Remove this commented-out code?

Reply:
Actually... I removed this because that's what I did for `leopard`, but actually we have test cases for `sb-edit`, so we might as well keep this part of the action.